packages/web-components/src/search/search-overlay.ts

```diff
@@ -0,0 +1,192 @@
+import { ProductResult, ProductSearchBuilder, ProductSearchResponse, SearchCollectionBuilder, SearchTermPredictionBuilder, SearchTermPredictionResponse, SearchTermPredictionResult } from '@relewise/client';
```

Comment:
Dette bør hedde `product-search-overlay` - Så vi også kan have et content overlay :)

Reply:
Jeg har nu rettet de componenter dette er relevant for til at heder `product-search-...`. 🙂